interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/mlmodel/'
+  );
+}])
```

Comment:
@mhuyck and @dongbohu : I actually like having this resource be separate from the components file, since it is not actually a component (in the Angular sense of the term "component"), and it has its own specific function - to communicate with the REST API. From a pragmatic point of view, I think it makes sense to keep this modules loosely coupled like this, especially if we end up needing to use this resource in other places.